addons/website_forum/static/src/js/website_forum.js

```diff
@@ -251,6 +251,7 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
             title: title,
             sticky: false,
             type: "warning",
+            messageIsHtml: true, // dynamic parts of the message are escaped above
```

Comment:
Why do you explain that dynamic parts of the message are escaped above while not escaping is done above? :thinking: